portal-kernel/src/com/liferay/portal/kernel/portlet/configuration/permission/propagation/BasePortletConfigurationPermissionPropagation.java

```diff
@@ -67,15 +67,17 @@ public boolean isAvailable(PortletRequest portletRequest) {
 	public void updatePermissionPropagation(
 		ActionRequest actionRequest, ActionResponse actionResponse) {
 
-		ThemeDisplay themeDisplay = (ThemeDisplay)actionRequest.getAttribute(
-			WebKeys.THEME_DISPLAY);
+		if (isAvailable(actionRequest)) {
+			ThemeDisplay themeDisplay =
+				(ThemeDisplay)actionRequest.getAttribute(WebKeys.THEME_DISPLAY);
 
-		PermissionPropagationEntryLocalServiceUtil.
-			updatePermissionPropagationEntry(
-				themeDisplay.getCompanyId(), getGroupId(actionRequest),
-				getClassName(), getClassPK(actionRequest),
-				ParamUtil.getBoolean(
-					actionRequest, "permissionPropagationEnabled"));
+			PermissionPropagationEntryLocalServiceUtil.
+				updatePermissionPropagationEntry(
+					themeDisplay.getCompanyId(), getGroupId(actionRequest),
+					getClassName(), getClassPK(actionRequest),
+					ParamUtil.getBoolean(
+						actionRequest, "permissionPropagationEnabled"));
+		}
```

Comment:
Is it better?
```
	if (!isAvailable(actionRequest)) {
		return;
	}

	ThemeDisplay themeDisplay =
			(ThemeDisplay)actionRequest.getAttribute(WebKeys.THEME_DISPLAY);

		PermissionPropagationEntryLocalServiceUtil.
			updatePermissionPropagationEntry(
				themeDisplay.getCompanyId(), getGroupId(actionRequest),
				getClassName(), getClassPK(actionRequest),
				ParamUtil.getBoolean(
					actionRequest, "permissionPropagationEnabled"));

```

Reply:
Thanks you,
It's not better, however, it will follow the LR code convention.